nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -233,53 +236,57 @@ Save your changes to your `nautobot_config.py` and then proceed to the next step
+??? note "MySQL Unicode Settings"
+
+    If you are using MySQL as your database backend, and you want to enable support for Unicode emojis, please make sure to add `"OPTIONS": {"charset": "utf8mb4"}` to your `DATABASES` setting. Please see the [configuration guide on MySQL Unicode settings](../configuration/required-settings.md#mysql-unicode-settings) for more information.
 
 ## Optional Settings
 
-All Python packages required by Nautobot will be installed automatically when running `pip3 install nautobot`.
+??? abstract "Optional Settings"
 
-Nautobot also supports the ability to install optional Python packages. If desired, these packages should be listed in `local_requirements.txt` within the `NAUTOBOT_ROOT` directory, such as `/opt/nautobot/local_requirements.txt`.
+    All Python packages required by Nautobot will be installed automatically when running `pip3 install nautobot`.
 
-If you decide to use any [Nautobot Apps](../../../apps/index.md), they should be listed in this file.
+    Nautobot also supports the ability to install optional Python packages. If desired, these packages should be listed in `local_requirements.txt` within the `NAUTOBOT_ROOT` directory, such as `/opt/nautobot/local_requirements.txt`.
 
-We will cover two examples of common optional settings below.
+    If you decide to use any [Nautobot Apps](../../../apps/index.md), they should be listed in this file.
 
-### Configuring NAPALM
+    We will cover two examples of common optional settings below.
 
-Nautobot provides built-in support for the [NAPALM automation](https://github.com/napalm-automation/napalm/) library, which allows Nautobot to fetch live data from devices and return it to a requester via its REST API. The [`NAPALM_USERNAME`](../configuration/optional-settings.md#napalm_username) and [`NAPALM_PASSWORD`](../configuration/optional-settings.md#napalm_password) configuration parameters define the credentials to be used when connecting to a device.
+    ### Configuring NAPALM
 
-To use NAPALM, add `nautobot[napalm]` to your `local_requirements.txt` so that it can be installed and kept up to date:
+    Nautobot provides built-in support for the [NAPALM automation](https://github.com/napalm-automation/napalm/) library, which allows Nautobot to fetch live data from devices and return it to a requester via its REST API. The [`NAPALM_USERNAME`](../configuration/optional-settings.md#napalm_username) and [`NAPALM_PASSWORD`](../configuration/optional-settings.md#napalm_password) configuration parameters define the credentials to be used when connecting to a device.
 
-```no-highlight
-echo "nautobot[napalm]" >> $NAUTOBOT_ROOT/local_requirements.txt
-```
+    To use NAPALM, add `nautobot[napalm]` to your `local_requirements.txt` so that it can be installed and kept up to date:
 
-### Remote File Storage
+    ```no-highlight title="Add napalm to the local_requirements.txt file"
+    echo "nautobot[napalm]" >> $NAUTOBOT_ROOT/local_requirements.txt
+    ```
```

Comment:
Not blocking for this PR by any means, but at some point I'd love for us to suggest using Poetry instead of pip to manage additional dependencies.